features/unit/abi_responsejsons/zeroLenStaticArrayArg.json

```
@@ -0,0 +1 @@
+{"name":"nonsense","args":[{"type":"(uint64,address)[0]"}],"returns":{"type":"(uint64,address)"}}
```

@ahangsu Checking my understanding:  Should the expected _name_ change based on https://github.com/algorand/algorand-sdk-testing/pull/250/commits/3f1fbd8d9c5b5e8037f008324aa9a7e21c62dc8e?

A more robust sanity check is to re-trigger a downstream SDK build + link successful build.


```suggestion
{"name":"zeroLenStaticArrayArg","args":[{"type":"(uint64,address)[0]"}],"returns":{"type":"(uint64,address)"}}
```

---

right, the change is correct, and I will locally confirm the change passes

---

Confirmed that the change passes in go-sdk on my local end

---

@ahangsu I trust you, but want to verify.  I kicked off one build:  https://app.circleci.com/pipelines/github/algorand/go-algorand-sdk/612/workflows/776b4e2d-c12f-45b2-a788-be1fb5ed32d1.  I'll approve after it completes.

---

got it, was wondering why go SDK CI is still running right now